src/swap/components/SwapProvider.tsx

```diff
@@ -276,6 +282,7 @@ export function SwapProvider({
         setLifeCycleStatus,
         swapTransaction: response,
         useAggregator,
+        maxSlippage,
```

Comment:
alphabetical order

Reply:
Also, I don't think is clean to just pass the `maxSlippage`. Like think 7 steps ahead, `statusData` will contain other info that need to persist inside `processSwapTransaction` when `setLifeCycleStatus` is called.

So we should pass `lifeCycleStatus`, inside `processSwapTransaction`.

So that we can persit the info we need from `statusData` when we recall `setLifeCycleStatus`.